lib/passage_flutter_platform/passage_flutter_platform_interface.dart

```diff
@@ -100,10 +100,6 @@ abstract class PassageFlutterPlatform extends PlatformInterface {
         'finishSocialAuthentication() has not been implemented.');
   }
 
-  Future<AuthResult> authorizeIOSWith(PassageSocialConnection connection) {
-    throw UnimplementedError('authorizeIOSWith() has not been implemented.');
-  }
-
```

Comment:
I don't think we can/should get rid of this method. The iOS method returns an AuthResult, while Android and web do not.

Reply:
how about `Future<AuthResult?> authorize()`? For iOS, It will return AuthResult, for Android and web will be null.

Reply:
I personally believe they should be separate methods, as they do somewhat different things and have different expected results, and one is async while the other is not (or at least _should_ not be).

Reply:
gotcha, makes sense!